apps/nameguard.io/src/components/organisms/Header.tsx

```diff
@@ -1,75 +1,74 @@
 "use client";
 
-import Link from "next/link";
+import NextLink from "next/link";
 
 import { GithubIcon, NGSearchIcon } from "@/components/atoms";
 import { NGSearch } from "@/components/molecules";
 import MobileMenu from "./MobileMenu";
 import { CalendarButton } from "../atoms";
-import { Button, IconButton } from "@namehash/namekit-react";
+import { Button, IconButton, Link } from "@namehash/namekit-react";
 
 export const Header = () => {
   return (
     <header className="sticky bg-white top-0 w-full z-20 border-b border-gray-300 h-[56px] py-[9px] sm:h-[70px] sm:py-4 select-none">
       <div className="max-w-7xl mx-auto items-center justify-between flex flex-row px-6">
         <div className="flex flex-row lg:gap-2 xl:gap-7 justify-between items-center">
           <div className="flex flex-row justify-between items-center gap-1 cursor-pointer">
-            <Link
+            <NextLink
```

Comment:
So the way `Link` was created initially was that it was just a wrapper on top of `a`, since it's not always a `NextLink` (that has additional helpers). For example, `Link` can be used inside of `nameguard-react` which isn't bound to Next.js

If we know it's an internal link, `NextLink` is a good use here because of the benefits that brings with prefetching.